cerbero/packages/debian.py

```diff
@@ -201,12 +201,19 @@ def create_tree(self, tmpdir):
         os.mkdir(os.path.join(packagedir, 'source'))
         m.action(_('Creating debian package structure at %s for package %s') %
                 (srcdir, self.package.name))
+        files_list = self.files_list(PackageType.RUNTIME)
+        files = '\n'.join([os.path.join(self.install_dir + '/' + f) \
+            for f in files_list])
         if os.path.exists(self.package.resources_postinstall):
-            shutil.copy(os.path.join(self.package.resources_postinstall),
-                        os.path.join(packagedir, 'postinst'))
+            script = open(self.package.resources_postinstall).read()
```

Comment:
For consistency I would use shell.replace, keeping the old file copy +  shell.replace